src/common/options/crimson.yaml.in

```diff
@@ -16,7 +16,14 @@ options:
   type: uint
   level: advanced
   desc: The number of threads for serving alienized ObjectStore
-  default: 6
+  default: 0
+  flags:
+  - startup
+- name: crimson_alien_op_num_threads_per_core
+  type: uint
+  level: advanced
+  desc: The number of threads (per core) for serving alienized ObjectStore
+  default: 4
```

Comment:
The existing `crimson_alien_op_num_threads` can already set the threads of alienstore workers.

`crimson_alien_op_num_threads_per_core` works as a suger to set it by multiplying `alien_cpu_set.size()`, but it might not be really useful (or even misleading) from my observations:

We tried to find the optimal configuration using different kinds of CPUs (different core numbers, different per-core performance), by tuning the core allocation ratio of seastar reactor vs alienstore, and the number of alienstore threads. But seems the configuration of `num-alien-op-threads` is independent to the `alien-cpu-set`. For example, in one platform, the ratio is 4, and in another platform, the ratio turns out to be 1. IIUC exposing an additional `crimson_alien_op_num_threads_per_core` might be misleading to suggest that there is a preferred default.

And I think `crimson_alien_op_num_threads_per_core (4)` is also misleading when there are multiple OSDs sharing the same `alien_cpu_set`, so that the actual running alienstore threads per core is 8 if there are 2 OSDs, and 16 when there are 4 OSDs.

In short, seems to me the original `crimson_alien_op_num_threads` is also sufficient for Option 1 (Optimal).

Reply:
I agree that "per-core" might be misleading. Thank you for the review, I'll go back to only having num_threads.

Reply:
### Option 1 (Optimal)
...
* ~Alienstore threads: `alien_cpu_set.size()` * `crimson_alien_op_num_threads_per_core` (default = 4).~
* Alienstore threads num: `crimson_alien_op_num_threads`

...